doc/ranger.pod

```diff
@@ -620,6 +620,47 @@ A key chain that allows you to quickly change the line mode of all the files of
 the current directory.  For a more permanent solution, use the command
 "default_linemode" in your rc.conf.
 
+=item .n
+
+Apply a new filename filter.
+
+=item .d
+
+Apply the typefilter "directory".
+
+=item .f
+
+Apply the typefilter "file".
+
+=item .l
+
+Apply the typefilter "symlink".
+
+=item .|
+
+Combine te two topmost filters from the filter stack in the "OR"
+relationship, instead of the "AND" used implicitly.
+
+=item .!
+
+Negate the topmost filter.
+
+=item .c
+
+Clear the filter stack.
+
+=item .*
```

Comment:
> Oh boy, won't this be hard to keep track of mentally?

In case of confusion, one can press `..` to view the filter stack state.

> What happens if you haven't used .! or .| or you used them way down the stack?

It's a nop then, it just does nothing as decomposing a simple filter returns a single-element list with this filter on its own. If they are way down the stack, you would need to make them topmost first.

Reply:
Hmm, do we need a stack rotate op? So you can decompose whichever filter you want?

Reply:
Good idea, I'll add it in a minute.

Reply:
Done, rotation added.